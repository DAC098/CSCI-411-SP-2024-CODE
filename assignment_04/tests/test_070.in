6
2 4 2
octysliswxfzvaxj wtdqnizvfaoddn
topupdokmndggncphdowboaedpepeiz smzbizrpbofvgdkbgp
agagaaaaaagagaaaa gatt
pggehnfyelfg jbwoeclbbfpuytdoxtiqknrkbc
eoxmxgtkuqfsbtp kudmwjshlsbzckfhecwvnyxuis
gtgggaatacca aaa
