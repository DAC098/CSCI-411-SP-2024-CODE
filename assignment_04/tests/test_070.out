octysliswxfzvaxj
_wtdqni_zvfaoddn
30
topupdokmndggncphdowboaedpepeiz
_______sm_zbizrp____bo_fvgdkbgp
80
agagaaaaaagagaaaa
____________g_att
56
_pgge__hnf__y_________elfg
jbwoeclbbfpuytdoxtiqknrkbc
46
eoxm______xgtk_____uqfsbtp
kudmwjshlsbzckfhecwvnyxuis
48
gtgggaatacca
______a_a__a
36
